### ISSUE TITLE
python layer for batchnorm

### ISSUE THREAD

**Issue by mingtop** [BVLC/caffe#3380]
I found there has no layer of batchnorm in python interface

**Comment by MosNicholas** [comment]
Has this been resolved?

**Comment by shelhamer** [comment]
Sorry, this doesn't make sense. All layers included in the Caffe library are exposed to pycaffe since the Python interface simply links to the main library and its registered layers. If you had an older Caffe from before the batch norm layers was included, you'd need to re-compile for the layer to be included.

Please ask usage questions on the mailing list.

From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:

> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
